src/docs/concepts-users-and-authentication/user-permissions/index.md

```diff
@@ -98,12 +110,160 @@ Now, you can check or uncheck the permissions checkbox for this user.
 
 ![User permissions checkbox](./user_publish_listings_checkbox.png)
 
+See which endpoints are affected by this setting
+[here](#restrict-publishing-rights).
+
+Read more about this feature in the
+[Help Center](https://www.sharetribe.com/help/en/articles/9503118-restrict-listing-publishing-rights).
+
+## Restrict transaction rights
+
+You might also want to restrict the ability to initiate transactions to
+specific users only. For example, if you have a marketplace where users
+can view listings but need to verify their identity before making
+purchases, you can enable this feature to manually control who can
+initiate transactions. Similarly, if your platform uses a subscription
+model, you may want to allow only paying users to start transactions. In
+some cases, you may run a marketplace that functions primarily as a
+showroom, where users can browse but cannot buy until the operator
+grants them permission.
+
+You can toggle the setting in the Access control tab.
+
+![Restrict transaction rights checkbox](./users_restrict_transaction_rights.png)
+
+Once this checkbox is selected, similiarly to how publishing rights are
+granted, you can see the permission status of each user in the Console's
+Manage > Users view.
+
+See which endpoints are affected by this setting
+[here](#restrict-transaction-rights-1).
+
+Read more about this feature in the
+[Help Center](https://www.sharetribe.com/help/en/articles/9790336).
+
+## API level restrictions
+
+Toggling permissions under the "Access Control" tab in the Console
+directly impacts API behavior. When permissions are modified, endpoint
+access is adjusted accordingly, ensuring that users cannot bypass the
+restrictions imposed by these settings. This guarantees access control
+policies are strictly enforced. These restrictions only apply in the
+Marketplace API.
```

Comment:
Would it make sense to add something here to give more context on why it's significant that these restrictions only apply in Marketplace API? Like
- marketplace end-users never access Integration API directly, so Integration API endpoints don't need to be similarly restricted
- if you create an integration where marketplace users indirectly use Integration API, you'll need to enforce the restrictions based on the currentUser effectivePermissionSet similarly to other custom dev
- if you are creating integrations that fetch or update information in Integration API related to third parties, you'd also need to consider how to apply these restrictions in your integration

Reply:
@SariSaar great points! Do you have an example of the last bullet point? I'm not entirely sure in what kind of scenario this would be relevant in.

Reply:
Something like
- you restrict publishing rights behind a paywall
- you want to allow users to post listings on behalf of other users (e.g. org accounts)
- => if someone's subscription has ended and you've revoked their publishing rights, you also want to prevent them from posting on behalf of others